plugins/modules/aci_file_remote_path.py

```diff
@@ -0,0 +1,347 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2023, Tim Cragg (@timcragg) <tcragg@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "certified"}
+
+DOCUMENTATION = r"""
+---
+module: aci_file_remote_path
+short_description: Manage Import/Export File Remote Paths (file:RemotePath)
+description:
+- Manage Import/Export File Remote Paths on Cisco ACI fabrics.
+options:
+  name:
+    description:
+    - The name of the File Remote Path.
+    type: str
+  description:
+    description:
+    - Description of the File Remote Path
+    type: str
+  remote_host:
+    description:
+    - Hostname or IP Address of the remote host
+    type: str
+  remote_port:
+    description:
+    - Port to access the remote host
+    type: int
+  remote_protocol:
+    description:
+    - Protocol to use to connect to the remote host
+    type: str
+    choices: [ ftp, scp, sftp ]
+  auth_type:
+    description:
+    - Authentication type for the remote host. Cannot be set to ssh_key if protocol is ftp
+    type: str
+    choices: [ password, ssh_key ]
+  remote_user:
```

Comment:
Should we prepend remote or should we remove that from the input to shorten input?